CHANGELOG.md

```diff
@@ -8,6 +8,30 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.3.5] - 2023-10-27
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.3.4...v1.3.5)
+
+### Added
+
+* Export metrics from MySQL and Postgres ([#1023](https://github.com/openfga/openfga/pull/1023))
```

Comment:
Could you amend this changelog and the release notes https://github.com/openfga/openfga/releases/tag/v1.3.5

to mention that to use this feature you need to turn on both `OPENFGA_METRICS_ENABLED` and `OPENFGA_DATASTORE_METRICS_ENABLED` ?